python/tskit/tables.py

```diff
@@ -2500,11 +2500,31 @@ def __len__(self):
 
 # TODO move to reference_sequence.py when we start adding more functionality.
 class ReferenceSequence(metadata.MetadataProvider):
+    """
+    The :ref:`reference sequence<sec_data_model_reference_sequence>` associated
+    with a given :class:`.TableCollection` or :class:`.TreeSequence`.
+
+    Metadata concerning reference sequences can be described using the
+    :attr:`.metadata_schema` and stored in the :attr:`.metadata` attribute.
+    See the :ref:`examples<sec_metadata_examples_reference_sequence>` for
+    idiomatic usage.
+
+    .. warning:: This API is preliminary and currently only supports accessing
+       reference sequence information via the ``.data`` attribute. Future versions
+       will also enable transparent fetching of known reference sequences
+       from a URL.
```

Comment:
Would e nice to link to an issue here so we can gauge demand.

Reply:
Done: #2022